doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -426,9 +426,9 @@ async def _start_build(
     @staticmethod
     async def get_installed_packages(
         image_pullspec: str, arches: list[str], registry_auth_file: Optional[str] = None
-    ) -> list:
+    ) -> tuple[list, list]:
```

Comment:
typing.Tuple[Set, Set] ?

Reply:
I want to disturbe the method fingerprint as little as possible, it was return a list, so I return another list now ...